nengo_viz/templates/page.html

```diff
@@ -15,21 +15,54 @@
     </head>
     <body onload="onload();">
         <div id="main"></div>
-        <div id="control"></div>
+
+        <!-- TODO: Is large z-index the method we want to use for keeping simcontrol on top  -->
+        <!-- or do we want to dynamically raise it when other elements have theirs increased -->
+        <div id="control">
+
+            <div class="time_slider">
+                <div class="shown_time"> 
+                    <!-- where are these used?  -->
+                    <div id="d1">  
```

Comment:
From what I can tell these aren't used anywhere, so I think we can take them out.